src/styles/variables.scss

```diff
@@ -3,18 +3,24 @@ $content-side-width: 260px;
 
 $orange: #ffcc00;
 $dark-orange: #ff9900;
-$dark-red: #CF3F3F;
+$dark-red: #f44336;
+$green: #8bc34a;
+$off-white: #fafafa;
+$white: #fcfcfc;
+$inactive-gray: #9ea1a3;
 $new: #68BD6A;
 $new-hover: #80ce81;
 
 $border-color: #ddd;
 
-$btn-initial: #b9b9b9;
+$btn-initial: #9ea1a3;
 $btn-initial-border: #828282;
+$btn-fat-hover: $dark-orange;
+$btn-fat-hover-border: $dark-orange;
 $btn-active: #ff6e00;
 $btn-active-border: #dd5c00;
-$btn-inactive: $btn-initial;
-$btn-inactive-border: $btn-initial-border;
+$btn-inactive: $btn-initial !important;
+$btn-inactive-border: $btn-initial-border !important;
 $btn-success: #8dcf3f;
```

Comment:
To make sure color doesn't change even if another attribute is added to parent element